groups/utils.js

```diff
@@ -178,6 +178,31 @@ function setParamValueInURL(paramKey, paramValue) {
   );
 }
 
+function fullDateString(timestamp) {
```

Comment:
Add validation to ensure the timestamp is valid to prevent invalid date 
move it from the usage to the util function